LF/character.js

```diff
@@ -1588,34 +1617,46 @@ define(['LF/livingobject', 'LF/global', 'core/combodec', 'core/util', 'LF/util']
         }
       }
 
-      $.injury(inj)
```

Comment:
is there a reason of moving this line?

Reply:
The offset_attack function will have return a delay if this line is NOT moved

Reply:
Thanks. Good to know